pkg/api/repositories.go

```diff
@@ -153,6 +153,15 @@ type RepositoryExportRequest struct {
 	RepositoryUuids []string `json:"repository_uuids" validate:"required"` // List of repository uuids to export
 }
 
+type RepositorySearchUploadsRequest struct {
+	Hashes []string `json:"sha256" validate:"required"`
+}
+
+type RepositorySearchUploadsResponse struct {
+	Found   []string `json:"found"`   // List of file hashes that were already uploaded
```

Comment:
We actually want the HREF of the artifact to be returned too, so it maybe it makes sense for found to be a hash:

```
found:   {
    $sha256:  $ARTIFACT_HREF
},
```

Reply:
okey dokey, wasn't in the ticket description, but makes sense, will add 🫡😅